core/bindings.rs

```diff
@@ -437,7 +442,12 @@ fn opcall_sync<'s>(
   {
     Ok(op_id) => op_id,
     Err(err) => {
-      throw_type_error(scope, format!("invalid op id: {}", err));
+      let msg = if args.get(0).is_undefined() {
+        UNDEFINED_OP_ID_MSG.to_string()
+      } else {
+        format!("invalid op id: {}", err)
+      };
+      throw_type_error(scope, msg);
```

Comment:
This is really just micro-optimization-feel-free-to-ignore territory but...:
```suggestion
      let msg = if args.get(0).is_undefined() {
        UNDEFINED_OP_ID_MSG
      } else {
        &format!("invalid op id: {}", err)
      };
      throw_type_error(scope, msg);
```
(ditto below)

Reply:
It's not a hot path, I'm gonna ignore it